pkg/gatewayserver/gatewayserver.go

```diff
@@ -661,6 +662,10 @@ func (gs *GatewayServer) handleUpstream(conn connectionEntry) {
 
 	hosts := make([]*upstreamHost, 0, len(gs.upstreamHandlers))
 	for name, handler := range gs.upstreamHandlers {
+		if name == "packetbroker" && gtw.DisablePacketBrokerForwarding {
+			logger.Info("Disable Packet Broker Forwarding")
```

Comment:
```suggestion
			logger.Info("Disable Packet Broker forwarding")
```

Reply:
Can we downgrade this to a `Debug` or not print this at all?